pymongo/srv_resolver.py

```diff
@@ -83,8 +85,8 @@ def get_options(self):
 
     def _resolve_uri(self, encapsulate_errors):
         try:
-            results = _resolve('_mongodb._tcp.' + self.__fqdn, 'SRV',
-                               lifetime=self.__connect_timeout)
+            results = _resolve('_'+str(self.__srv)+'._tcp.' + self.__fqdn,
```

Comment:
Add spaces around the '+' characters. Also, why do you have to wrap self.__srv in str? Is it being converted to a bytes before it gets here?

Reply:
I think you are right and we can remove that.